### PR TITLE
Use symmetric encryption for environment variable

### LINKED ISSUE #437: Replace box env encryption with crypto/symmetric.go
See #436 

The (bad 😢) box implementation: https://github.com/ubclaunchpad/inertia/blob/master/daemon/inertiad/crypto/box.go

### PR DESCRIPTION
**Pull request by dreamer-89**
:tickets: **Ticket(s)**: Closes #437 

---

## :construction_worker: Changes

A brief summary of what changes were introduced.

## :flashlight: Testing Instructions

Explain how to test your changes, if applicable.

### CODE REVIEW

File: daemon/inertiad/project/data.go [hunk 1]
```diff
@@ -41,12 +39,12 @@ func newDataManager(dbPath string) (*DeploymentDataManager, error) {
 		return nil, err
 	}
 
-	encryptPublicKey, encryptPrivateKey, decryptPublicKey, decryptPrivateKey, err := crypto.GenerateKeys()
-
+	// encryptPublicKey, encryptPrivateKey, decryptPublicKey, decryptPrivateKey, err := crypto.GenerateKeys()
+	key := make([]byte, 32)
+	_, err = gcrypto.Read(key)
```
Comment (bobheadxi) [inline daemon/inertiad/project/data.go]: you'll need an err check here! like so:

```go
if _, err = grcypto.Read(key); err != nil {
  return fmt.Errorf("failed to generate key: %s")
}
```

File: daemon/inertiad/project/data.go [hunk 2]
```diff
@@ -41,12 +39,12 @@ func newDataManager(dbPath string) (*DeploymentDataManager, error) {
 		return nil, err
 	}
 
-	encryptPublicKey, encryptPrivateKey, decryptPublicKey, decryptPrivateKey, err := crypto.GenerateKeys()
-
+	// encryptPublicKey, encryptPrivateKey, decryptPublicKey, decryptPrivateKey, err := crypto.GenerateKeys()
```
Comment (bobheadxi) [inline daemon/inertiad/project/data.go]:
```suggestion
```

File: daemon/inertiad/project/data.go [hunk 3]
```diff
@@ -4,6 +4,8 @@ import (
 	"encoding/json"
 	"errors"
 
+	gcrypto "crypto/rand"
```
Comment (bobheadxi) [inline daemon/inertiad/project/data.go]: might want to leave this as `rand` - there's no need to rename it
```suggestion
	"crypto/rand"
```

**Comment by dreamer-89** [comment]
Seems to be some issue with TestBootstrapIntegration unit/integration test. Travis check failing because of this.

**Comment by bobheadxi** [comment]
@dreamer-89 unfortunately thats currently expected - I'm having an issue with bootstrapping suddenly breaking a little while ago. I haven't had time to really investigate and get it fixed up, so for now we're just ignoring that particular test (context: #447 )

**Pull request merged**